interpreter/exec/int.ml

```diff
@@ -173,6 +174,8 @@ struct
     let n = clamp_rotate_count y in
     or_ (Rep.shift_right_logical x n) (Rep.shift_left x (Rep.bitwidth - n))
 
+  let xchg x y = y
```

Comment:
Shouldn't this return x?

Reply:
No, we want to return the value to store, which is the new value, `y`. It does seem a bit weird to have in int.ml, though, so I moved in inline to `rmwop`.